src/views/TeamPage.vue

```diff
@@ -1,244 +1,27 @@
-          <router-link
-            :data-test="`team-member-${member}-page-link`"
-            :to="{ name: 'TeamMember', params: { member } }"
-          >
-            <!-- TODO: Transform this into a component -->
-            <div
-              class="blog-card"
-              data-test="team-card"
-              :data-test-member-name="`team-member-${member}`"
-            >
-              <div class="meta">
-                <div
-                  :data-test="`team-member-${member}-index-photo`"
-                  class="photo"
-                  :style="`background-image: url( ${$t(`team.${member}.image`)} );`"
-                />
-              </div>
-              <div class="description">
-                <h2
-                  :data-test="`team-member-${member}-name`"
-                >
-                  {{ $t(`team.${member}.name`) }}
-                </h2>
-                <div class="socialIcons">
-                  <a
-                    v-if="$t(`team.${member}.github_url`).length > 1"
-                    :data-test="`team-member-${member}-github`"
-                    :href="$t(`team.${member}.github_url`)" target="_blank"
-                  >
-                    <i class="mobile-menu-icon fab fa-github" />
-                  </a>
-                  <a
-                    v-if="$t(`team.${member}.linkedin_url`).length > 1"
-                    :data-test="`team-member-${member}-linkedin`"
-                    :href="$t(`team.${member}.linkedin_url`)" target="_blank"
-                  >
-                    <i class="mobile-menu-icon fab fa-linkedin" />
-                  </a>
-                  <a
-                    v-if="$t(`team.${member}.twitter_url`).length > 1"
-                    :data-test="`team-member-${member}-twitter`"
-                    :href="$t(`team.${member}.twitter_url`)" target="_blank"
-                  >
-                    <i class="mobile-menu-icon fab fa-twitter" />
-                  </a>
-                  <a
-                    v-if="$t(`team.${member}.website`).length > 1"
-                    :data-test="`team-member-${member}-website`"
-                    :href="$t(`team.${member}.website`)" target="_blank"
-                  >
-                    <i class="mobile-menu-icon fa fa-cloud" />
-                  </a>
-                </div>
-
-                <router-link
-                  class="user-profile-link"
-                  :data-test="`team-member-${member}-page-link`"
-                  :to="{ name: 'TeamMember', params: { member } }"
-                >
-                  {{ t('redirect.profile') }}
-                </router-link>
-              </div>
-            </div>
-          </router-link>
-        </div>
-      </div>
+  <div class="container">
+    <h1 class="max-w-700px m-auto text-center text-2rem font-700" data-test="team-page-headline">
+      Schrödinger Hat's fam
+    </h1>
+    <div class="grid grid-cols-1 sm:grid-cols-2 lg:grid-cols-3 gap-20px p-1.5rem">
```

Comment:
Specific breakpoints rules `sm:*`, `lg:*` should be at the end of the string so we can quickly identify where they are 

So this would be:
```html
<div class="grid grid-cols-1 gap-20px p-1.5rem sm:grid-cols-2 lg:grid-cols-3">
```

Reply:
[fix(views): align unocss classes and position to project standard](https://github.com/Schrodinger-Hat/schrodinger-hat-website/pull/94/commits/ed76e4b7ed24d493e6efbe4bb4c35d46d9c39855)
fixed